collectors/statsd.plugin/README.md

```diff
@@ -38,53 +40,63 @@ Netdata fully supports the statsd protocol. All statsd client libraries can be u
 
-     For counters use `|c` (etsy/statsd compatible) or `|C` (brubeck compatible), for meters use `|m`.
-
-     Sampling rate is supported (check below).
+     - Counters use `|c` (etsy/statsd compatible) or `|C` (brubeck compatible)
+     - Meters use `|m`
 
-     When a counter or meter is not collected and the setting is not to show gaps on the charts (the default), zero will be shown, until a data collection event changes it.
+     [Sampling rate](#sampling-rates) is supported.
 
+     When a counter or meter is not collected, Netdata **defaults** to showing a zero value, until a data collection event changes the value.
+     
 -   **Timers** and **Histograms**
 
      The application sends `name:value|ms` or `name:value|h`, where `value` is any **decimal/fractional** number, statsd reports **min**, **max**, **average**, **sum**, **95th percentile**, **median** and **standard deviation** and the total number of times it was updated (events).
 
-     For timers use `|ms`, or histograms use `|h`. The only difference between the two, is the `units` of the charts (timers report milliseconds).
+     - Timers use `|ms`
+     - Histograms use `|h`
+  
+     The only difference between the two, is the `units` of the charts, as timers report *miliseconds*.
 
-     Sampling rate is supported (check below).
+     [Sampling rate](#sampling-rates) is supported.
 
-     When a timer or histogram is not collected and the setting is not to show gaps on the charts (the default), zero will be shown, until a data collection event changes it.
+     When a counter or meter is not collected, Netdata **defaults** to showing a zero value, until a data collection event changes the value.
 
 -   **Sets**
 
       The application sends `name:value|s`, where `value` is anything (**number or text**, leading and trailing spaces are removed), statsd reports the number of unique values sent and the number of times it was updated (events).
 
      Sampling rate is **not** supported for Sets. `value` is always considered text.
 
-     When a set is not collected and the setting is not to show gaps on the charts (the default), zero will be shown, until a data collection event changes it.
+     When a counter or meter is not collected, Netdata **defaults** to showing a zero value, until a data collection event changes the value.
 
 #### Sampling Rates
 
-The application may append `|@sampling_rate`, where `sampling_rate` is a number from `0.0` to `1.0`, to have statsd extrapolate the value, to predict to total for the whole period. So, if the application reports to statsd a value for 1/10th of the time, it can append `|@0.1` to the metrics it sends to statsd.
+The application may append `|@sampling_rate`, where `sampling_rate` is a number from `0.0` to `1.0` in order for StatD to extrapolate the value and predict the total for the entire period. If the application reports to StatsD a value for 1/10th of the time, it can append `|@0.1` to the metrics it sends to statsd.
```

Comment:
```suggestion
The application may append `|@sampling_rate`, where `sampling_rate` is a number from `0.0` to `1.0` in order for StatsD to extrapolate the value and predict the total for the entire period. If the application reports to StatsD a value for 1/10th of the time, it can append `|@0.1` to the metrics it sends to statsd.
```